openfpga_flow/tasks/fpga_verilog/synthesizable_verilog/config/task.conf

```diff
@@ -34,3 +34,27 @@ bench0_chan_width = 300
 
 [SCRIPT_PARAM_MIN_ROUTE_CHAN_WIDTH]
 end_flow_with_test=
+
```

Comment:
This should work well, you might need to add  `end_flow_with_test` in each script param